src/main/java/com/inscopemetrics/mad/sinks/TelemetrySink.java

```diff
@@ -84,13 +96,25 @@ public Builder setActorSystem(final ActorSystem value) {
         }
 
         /**
-         * Sets the <code>Telemetry</code> actor path. Optional. Cannot be null or empty. "/users/telemetry" by default.
+         * Sets the metrics factory. Required. Cannot be null. Injected by default.
          *
-         * @param value the path to the <code>Telemetry</code> actor
+         * @param value the metrics factory
          * @return this builder
          */
-        public Builder setTelemetryActorPath(final String value) {
-            _telemetryActorPath = value;
+        public Builder setMetricsFactory(final MetricsFactory value) {
+            _metricsFactory = value;
+            return self();
+        }
+
+        /**
+         * Sets the set of statistics to send in lieu of a histogram. Optional.
+         * Cannot be null. Empty set by default.
+         *
+         * @param value the set of statistics to send in lieu of histogram
+         * @return this builder
+         */
+        public Builder setHistogramStatistics(final ImmutableSet<Statistic> value) {
```

Comment:
You only get you what you request or what is a dependency. The histogram statistic is a dependency of any explicit tpX statistics (since they are lifted from its calculator exactly as I do here). Since I don't want to compute and send tpX statistics anywhere, and only do for telemetry because it doesn't yet support histogram, this seems like more than right direction than computing explicit tpX statistics in MAD.

Reply:
Agreed.  Just didn't see the direction before.